.github/dependabot.yaml

```diff
@@ -2,7 +2,12 @@
 version: 2
 
```

Comment:
Just curious, why no package-ecosystem: pub?

Reply:
I ran into some issues when eval'ing it when it was in beta (https://github.com/dependabot/dependabot-core/issues/4979), and was generally waiting for them to be addressed before using the functionality more widely.